salt/modules/network.py

```diff
@@ -1333,6 +1333,60 @@ def ip_addrs6(interface=None, include_loopback=False, cidr=None):
 ipaddrs6 = salt.utils.functools.alias_function(ip_addrs6, "ipaddrs6")
 
 
+def ip_neighs():
+    """
+    Return the ip neighbour (arp) table from the minion for IPv4 addresses
+
+    .. versionadded:: 3007
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' network.ip_neighs
+    """
+    ret = {}
+    out = __salt__["cmd.run"]("ip neigh show")
+    for line in out.splitlines():
+        comps = line.split()
+        if len(comps) < 5:
+            continue
+        if "." in comps[0]:
+            ret[comps[4]] = comps[0]
+
+    return ret
+
+
+ipneighs = salt.utils.functools.alias_function(ip_neighs, "ipneighs")
```

Comment:
```suggestion
```
If this function is new, why support 2 names?

I'm guessing the existing aliases were to be backwards compatible with something?

Reply:
Not being backwards compatible with anything but maintaining existing convention, as stated in the PR, similar to ip_addrs and ip_addrs6, if we split it up for ```ip addr```, keeping it similar for ```ip neighbour``` , noting ```man 8 ip-neighbour``` shows 'ip neigh'

Reply:
ok